### ISSUE TITLE
jupiterbroadcasting.net - security warning

### ISSUE THREAD

**Issue by gerbrent** [JupiterBroadcasting/jupiterbroadcasting.com#284]
@StefanS-O this one is for you

w the new change to [new.jupiterbroadcasting.com](https://new.jupiterbroadcasting.com/) for the testing site, and with redirecting [jupiterbroadcasting.net](https://jupiterbroadcasting.net/) to said server, there is now presented a security error (on Firefox). 

As JB.net is still on much of our documentation and many of our podcast episodes, it would be great to solve this hiccup to redirect seemlessly to new.jb.com

thoughts @ironicbadger ?

![image](https://user-images.githubusercontent.com/7018928/184650490-1abb0da7-b95a-4350-bcd2-a5d9749bc1b6.png)

**Comment by reclaimingmytime** [comment]
You can continue through the security warning, which shows this page.
![image](https://user-images.githubusercontent.com/35609464/184652208-f8ba6daa-15d8-42f9-9335-472d09b08059.png)

Same for https://jb-live.jupiterbroadcasting.net/

**Comment by ironicbadger** [comment]
I don't have permissions delegated for jb.net like I do for .com. 

We need a full redirect. We are picking up the traefik default cert on the prod VPS but because the headers don't match any sites it has a rule for we're getting this error. Not much I can do tbh.

**Comment by StefanS-O** [comment]
I mean i pointed the DNS of the Domain to the IP of the new site, so probably just missing the vhost / letsencrypt there.
I could also take back those changes and do a redirect to new.jupiterbroadcasting.com, but i thought that was the proper way.

**Comment by ironicbadger** [comment]
> I mean i pointed the DNS of the Domain to the IP of the new site, so probably just missing the vhost / letsencrypt there. I could also take back those changes and do a redirect to new.jupiterbroadcasting.com, but i thought that was the proper way.

A redirect would be better please.

**Comment by StefanS-O** [comment]
Alright

**Comment by StefanS-O** [comment]
do you have setup jb-live somewhere, or just the main site?

**Comment by ironicbadger** [comment]
@gerbrent ?

**Comment by gerbrent** [comment]
really, I think JB Live should be part of the main site (#233 ) - unless there is some technical advantage to keep them seperate which isn't obvious to me.

**Comment by gerbrent** [comment]
> do you have setup jb-live somewhere, or just the main site?

jb-live is currently living alongside the main site, deployed in concert.

**Comment by StefanS-O** [comment]
Ok what i did:

1. www.jupiterbroadcasting.net and jupiterbroadcasting.net will redirect to https://new.jupiterbroadcasting.com/ preserving the request uri
2. jb-live.jupiterbroadcasting.net pointing to my server for now

DNS changes only have a 60 second TTL, so it should work soon

**Comment by elreydetoda** [comment]
For this:

>  1. www.jupiterbroadcasting.net and jupiterbroadcasting.net will redirect to https://new.jupiterbroadcasting.com/ preserving the request uri

Once we finish up migrating things I can work with someone to create a permanent redirector in a serverless function, which should only cost fractions of a penny per run. I did that for my companies blog when we transitioned from `blog. `sub-domain to using  `.com/blog`

For example: https://blog.secureideas.com/2018/09/automating-red-team-homelabs-part-1-kali-automation.html

That way @StefanS-O or JB doesn't have to keep running nginx forever to keep that redirector. Or alternatively traefik should be able to handle it. Whatever you all are comfortable with 😁
